tests/test_project/tests/test_qa.py

```diff
@@ -44,5 +47,35 @@ def test_qa_call_check_migration_name_failure(self):
                 else:
                     self.fail('SystemExit or Exception not raised')
 
+    def test_qa_call_check_commit_message_pass(self):
+        options = [
+            'commitcheck', '--token', self._gh_token,
+            '--commit', self._commit,
+            '--repo', self._repo
+        ]
+        with patch('argparse._sys.argv', options):
+            check_commit_message()
+
+    def test_qa_call_check_commit_message_failure(self):
+        options = [
+            [
+                'commitcheck', '--token', self._gh_token,
+                '--commit', self._commit,
```

Comment:
it's ok to hardcode values here, you should test all the failure cases that we mentioned in the discussion